docs/installations/build-openms-from-source.md

```diff
@@ -0,0 +1,10 @@
+Build OpenMS From Source
```

Comment:
Perhaps we could just link to build from source instructions in the OS specific pages and get rid of this section.